src/Platform/src/Platform.Windows.cpp

```diff
@@ -231,6 +232,47 @@ namespace OpenLoco::Platform
 
         return true;
     }
+
+    std::vector<std::string> getCmdLineVector(int, const char**)
+    {
+        // Discard the input as it will either be ASCII or not available
+
+        // We need to get the Wide command line and convert it to utf8 command line parameters
+        const auto cmdline = GetCommandLineW();
+
+        int argc{};
+        auto* argw = CommandLineToArgvW(cmdline, &argc);
+        if (argw == nullptr)
+        {
+            std::cerr << "CommandLineToArgvW failed to convert commandline";
```

Comment:
should we be using the Logging library here?

Reply:
I had to remove it and switch to iostreams as diagnostic lib depends on platform.